pkgs/dominikh/go-tools/staticcheck/registry.yaml

```diff
@@ -4,19 +4,21 @@ packages:
     repo_owner: dominikh
     repo_name: go-tools
     description: Go static analysis, detecting bugs, performance issues, and much more
-    asset: staticcheck_{{.OS}}_{{.Arch}}.tar.gz
+    asset: staticcheck_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
+    rosetta2: true
     files:
       - name: staticcheck
         src: staticcheck/staticcheck
     supported_envs:
-      - darwin
+      - darwin/amd64
```

Comment:
darwin/arm64 is supported by `rosetta2: true`.
Basically, we don't use `darwin/amd64`.